backend/FastAPI/main.py

```diff
@@ -1,33 +1,34 @@
+from setup import write_client, read_api, ORG, PARAMETERS
+from classes import Data, Dates
+from functions import get_query, list_all_items
 from fastapi.middleware.cors import CORSMiddleware
-from fastapi import FastAPI, Request
-from pandas import read_json, Timestamp
-from pydantic import BaseModel
-from influxdb_client_3 import InfluxDBClient3
-from influxdb_client import InfluxDBClient
+from fastapi import FastAPI
+from pandas import read_json
 from uvicorn import run
-from dotenv import load_dotenv
-from os import getenv
 from io import StringIO
 
-load_dotenv()
-
-## TIME FILTER FORMAT: 2023-11-15T12:00:00.000Z
-
-# -----------Constants-----------
-INFLUXDB_URL = getenv("FASTAPI_INFLUXDB_URL")
-TOKEN = getenv("FASTAPI_TOKEN")
-ORG = getenv("FASTAPI_ORG")
-BUCKET = getenv("FASTAPI_BUCKET")
-BASE_QUERY = f"""from(bucket: "{BUCKET}")"""
-
-# -----------InfluxDB-settings-----------
-read_client = InfluxDBClient(url=INFLUXDB_URL, token=TOKEN, org=ORG)
-write_client = InfluxDBClient3(host=INFLUXDB_URL, token=TOKEN, org=ORG, database=BUCKET)
-read_api = read_client.query_api()
-
+## TIME FILTER FORMAT FOR REQUEST: 2023-11-15T12:00:00.00
 
 # -----------App-settings-----------
-app = FastAPI()
+app = FastAPI(
+    title="AIrsight",
+    summary="AIrsight API",
+    version="1.0.0",
+    openapi_tags=[
+        {
+            "name": "Add item",
+            "description": "Add new item to database. Data must be in body.",
+        },
+        {
+            "name": "Latest data",
+            "description": "Get latest data from all items.",
+        },
+        {
+            "name": "Specific data (with timestamps)",
+            "description": "Get data by param from specific item with or withouth timestamps. Timestamps must be in body.",
```

Comment:
- Without*
- Same note as before with the request body line.
- What parameters?